### PR TITLE
Update privacy policy page to include section for Beamer

### LINKED ISSUE #20: Add Beamer to privacy section
## Overview

Adding Beamer as a service requires listing it under https://gnosis-safe.io/privacy/ 

## Requirements

adding the following to section 4:

```
**4.19 Beamer**

We use Beamer (https://www.getbeamer.com/) for notifications.

Beamer's purpose and function are further explained under the following link https://www.getbeamer.com/showcase/notification-center.

We do not store any information collected by Beamer, Beamer's privacy policy is available at https://www.getbeamer.com/privacy-policy/.
```

## Screens
 - Figma:
 - Zeplin:

### PR DESCRIPTION
**Pull request by usame-algan**
Resolves #20

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,4 @@ public
 npm-debug.log*
 
 .vscode
+.idea
```
Comment (iamacook) [inline .gitignore:25]: Why did you add this?

Comment (usame-algan) [reply .gitignore:25]: Its the webstorm project folder, similar to `.vscode` above.

**Review by iamacook — APPROVED** [review]

**Review by DiogoSoaress — APPROVED** [review]
